hasher-matcher-actioner/hmalib/dto.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+import typing as t
+
+# Not enforceable because named tuples can't have multiple inheritance, but all
+# DTO classes in this module should implement methods
+# `to_dynamodb_item(self)`
+# and `to_sqs_message(self)`
+
+
+class PDQHashRecord(t.NamedTuple):
+    """Successful execution at the hasher produces this record.
+    """
+
+    content_key: str
+    content_hash: str
+    quality: int
+    timestamp: datetime.datetime  # ISO-8601 formatted
+
+    def to_dynamodb_item(self) -> dict:
```

Comment:
Don't use dict, use t.Dict!

mypy strict will also force to type it-> t.Dict[str, ...]